### PR TITLE
Update the docs preview process

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request adds more detail to the docs preview process.

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,13 +29,28 @@ If you use VS Code, use the [Redocly VS Code extension](https://marketplace.visu
...
-- `npm run serve-reports` - preview Reports API doc
-- `npm run serve-storefront` - preview Storefront API docs
-- `npm run serve-combined` - preview combined API docs
+This process describes how to start a development server docs preview for the Rebilly API docs. 
+Changes made to the API specification while the development server is running are immediately displayed in the docs preview.
+
+Rebilly uses [Redocly](https://redocly.com/) to generate, manage, and preview API docs.
+
+1. In a terminal, install the [Redocly CLI](https://github.com/Redocly/redocly-cli). \
+   Use this command to install the package globally on your machine: `sudo npm install @redocly/cli -g`
```
Comment (adamaltman) [inline README.md:38]:
```suggestion
   Use this command to install the package globally on your machine: `npm install @redocly/cli -g`
```

File: README.md [hunk 2]
```diff
@@ -29,13 +29,28 @@ If you use VS Code, use the [Redocly VS Code extension](https://marketplace.visu
...
+1. [Sign in or create a Redocly account](https://app.redocly.com/signup). \
+   If you are a Rebilly employee, sign in using SSO and your Rebilly email address.
+1. In Redocly, in the top right of the screen, click your name, then click **My profile**.
+1. In the **Personal API keys** section, click **Add API key**.
+1. Enter an API key name, then click **Save**.
+1. Copy the API key.
+1. In a terminal, execute the following command: `openapi login`.
```
Comment (adamaltman) [inline README.md:45]:
```suggestion
1. In a terminal, execute the following command: `redocly login`. 
```

File: README.md [hunk 3]
```diff
@@ -29,13 +29,28 @@ If you use VS Code, use the [Redocly VS Code extension](https://marketplace.visu
...
+1. Enter an API key name, then click **Save**.
+1. Copy the API key.
+1. In a terminal, execute the following command: `openapi login`. 
+1. When prompted, paste your API key.
+1. Execute one of the following commands to start a development server docs preview:
+    - Core API docs: `npm run serve-core`
+    - Users API docs: `npm run serve-users`
+    - Reports API docs: `npm run serve-reports`
+    - Storefront API docs: `npm run serve-storefront`
+    - Combined API docs: `npm run serve-combined`
```
Comment (adamaltman) [inline README.md:52]:
```suggestion
    - Core API docs: `redocly preview-docs core@latest`
    - All API docs: `redocly preview-docs all@latest`
```

**Review by adamaltman — COMMENTED** [review]
A few updates.

**Review by adamaltman — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**